components/nexus-security/src/main/java/org/sonatype/nexus/security/WebSecurityModule.java

```diff
@@ -56,7 +64,7 @@
  *
  * @since 2.6.1
  *
- * @see SecurityModule
+ * @see TestSecurityModule
```

Comment:
I'll be removing it in my PR, so feel free to go ahead and merge this as-is and I'll clean it up later.